ci/utils/nbtest.sh

```diff
@@ -20,31 +21,30 @@ get_ipython().run_cell_magic=my_run_cell_magic
 NO_COLORS=--colors=NoColor
 EXITCODE=0
 NBTMPDIR="$WORKSPACE/tmp"
-mkdir -p ${NBTMPDIR}
+mkdir -p "${NBTMPDIR}"
 
-for nb in $*; do
-    NBFILENAME=$1
+for nb in "$@"; do
+    NBFILENAME=$nb
```

Comment:
This bug is copy-pasted everywhere, but I think doesn't impact anything since this script is usually called on individual notebooks.